frontend/src/components/modals/SubtitleToolsModal.tsx

```diff
@@ -106,6 +114,7 @@ const SubtitleToolView: FunctionComponent<SubtitleToolViewProps> = ({
         canSelect={CanSelectSubtitle}
         data={data}
       ></SimpleTable>
+      <Space h="400px"></Space>
```

Comment:
This has to change every time a new item is added or removed, i think the workaround is not defensive enough. What we can do is probably make the Modal `overflow: visible` and then adjust the Menu z-index to be displayed outside of the Model.

Reply:
Are you able to draft a PR around this? Is it a big deal or not? Thanks BTW :-)

Reply:
I will take a quick look later and reply here.

Reply:
I opened the #2455 with the proper fix. Please close.